islandora_openseadragon.install

```diff
@@ -38,3 +54,21 @@ function islandora_openseadragon_uninstall() {
   );
   array_walk($variables, 'variable_del');
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Add the required table for handling authentication tokens.
+ * This is the first instance that has this table.
+ */
+function islandora_openseadragon_update_7001(array &$sandbox) {
```

Comment:
Not too concerned, but [strictly, this should probably be `7100`; `70xx` is supposed to be for migrating from Drupal 6](https://api.drupal.org/api/drupal/modules!system!system.api.php/function/hook_update_N/7.x).